polyphemus/spec/fixtures/redcap_test2_metadata.json

```diff
@@ -0,0 +1,122 @@
+[
+  {
+    "field_name": "company_name",
+    "form_name": "cars",
+    "section_header": "",
+    "field_type": "text",
+    "field_label": "Company Name",
+    "select_choices_or_calculations": "",
+    "field_note": "",
+    "text_validation_type_or_show_slider_number": "",
+    "text_validation_min": "",
+    "text_validation_max": "",
+    "identifier": "",
+    "branching_logic": "",
+    "required_field": "",
+    "custom_alignment": "",
+    "question_number": "",
+    "matrix_group_name": "",
+    "matrix_ranking": "",
+    "field_annotation": ""
+  },
+  {
+    "field_name": "dof",
+    "form_name": "cars",
+    "section_header": "",
+    "field_type": "text",
+    "field_label": "date of founding",
+    "select_choices_or_calculations": "",
+    "field_note": "",
+    "text_validation_type_or_show_slider_number": "",
+    "text_validation_min": "",
+    "text_validation_max": "",
+    "identifier": "",
+    "branching_logic": "",
+    "required_field": "",
+    "custom_alignment": "",
+    "question_number": "",
+    "matrix_group_name": "",
+    "matrix_ranking": "",
+    "field_annotation": ""
+  },
+  {
+    "field_name": "car_class",
+    "form_name": "cars",
+    "section_header": "",
+    "field_type": "text",
+    "field_label": "car class",
+    "select_choices_or_calculations": "",
+    "field_note": "",
+    "text_validation_type_or_show_slider_number": "",
+    "text_validation_min": "",
+    "text_validation_max": "",
+    "identifier": "",
+    "branching_logic": "",
+    "required_field": "",
+    "custom_alignment": "",
+    "question_number": "",
+    "matrix_group_name": "",
+    "matrix_ranking": "",
+    "field_annotation": ""
+  },
+  {
+    "field_name": "year",
+    "form_name": "cars",
+    "section_header": "",
+    "field_type": "date",
+    "field_label": "calendar year",
+    "select_choices_or_calculations": "",
+    "field_note": "",
+    "text_validation_type_or_show_slider_number": "",
+    "text_validation_min": "",
+    "text_validation_max": "",
+    "identifier": "",
+    "branching_logic": "",
+    "required_field": "",
+    "custom_alignment": "",
+    "question_number": "",
+    "matrix_group_name": "",
+    "matrix_ranking": "",
+    "field_annotation": ""
+  },
+  {
+    "field_name":"feature",
+    "form_name":"cars",
+    "section_header":"",
+    "field_type":"checkbox",
+    "field_label":"Features",
+    "select_choices_or_calculations":"0, Window | 1, Door | 2, Spoiler | 3, Radio | 4, Engine | 5, Carbuerator | 6, Wheels",
```

Comment:
Is this what the checkbox output looks like, from REDCap?

Reply:
This is not the output, but the metadata for what the checkbox values are. The output will then be like `{ "field_name": "checkbox_field", "value": 0 }, { "field_name": "checkbox_field": "value": 1 }`, i.e. several records/values per field_name.